ert_shared/services/_webviz_ert_main.py

```diff
@@ -23,15 +24,34 @@ def handle_exit(*args: Any) -> None:  # pylint: disable=unused-argument)
 
 
 def create_config(
-    project_identifier: Optional[str], config_file: pathlib.Path, temp_config: Any
+    project_identifier: Optional[str],
+    config_file: pathlib.Path,
+    temp_config: Any,
+    experimental_mode: bool,
 ) -> None:
     with open(config_file, "r") as f:
         config_dict = yaml.safe_load(f)
         for page in config_dict["pages"]:
             for element in page["content"]:
                 for key in element:
                     element[key] = {"project_identifier": project_identifier}
-    output_str = yaml.dump(config_dict)
+
+    new_config_dict = config_dict
+
+    def filter_experimental(e, b: bool) -> bool:
```

Comment:
Is it possible to make `e` and `b` a bit more explicit something like `page` and `experimental_mode` 

also if `b` gets a type (`b: bool`) maybe also e should get a type just for consistency (`e: Dict[str, Any]`)  I am also ok with no variable types for these inner functions